extension/app/src/lib/types.ts

```diff
@@ -0,0 +1,12 @@
+import type { UploadedContentFragmentType } from "@dust-tt/client";
+
+export type UploadedFileKind = "attachment" | "tab_content";
```

Comment:
Should we have another `kind` for when the user just sends the selection? 
- `attachment` is when I attach a file
- `tab_content` is when I attach a full tab, using the extractor that keeps the hierarchy of the page.
- `tab_selection` is when I select some text, click right and send to Dust (in this case it's only the text that we send).

Reply:
+1  tab_selection types should not be superseded

Reply:
Indeed ! Didn't know about selection,  created a separate kind which isn't superseded 👍